build.yaml

```diff
@@ -586,6 +605,110 @@ common_cli:
     parse_logs vagrant box add --force "clusterhq/flocker-tutorial" \
       vagrant/tutorial/flocker-tutorial-\${FLOCKER_VERSION}.box
 
+  do_not_abort_on_errors: &do_not_abort_on_errors |
+    # make sure we don't abort the job on the first error we find
+    set +e
+
+  new_vagrantfile_for_yosemite: &new_vagrantfile_for_yosemite |
```

Comment:
:grey_question: Optional, here and elsewhere. Consider saying `osx_10_10` instead of `yosemite`, or at the least, saying `osx_yosemite`. 

Reasons:
* not everyone is familiar w/ OS X code names
* using `osx` everywhere makes it easy to grep for mac stuff
* using version numbers makes it easier to see what's older & can thus be deleted

Up to you though.